src/Components/benchmarkapps/Wasm.Performance/TestApp/wwwroot/benchmarks/index.js

```diff
@@ -11,15 +11,14 @@ new HtmlUI('E2E Performance', '#display');
 if (location.href.indexOf('#automated') !== -1) {
   (async function() {
     const query = new URLSearchParams(window.location.search);
-    const group = query.get('group');
```

Comment:
Can you explain what this change allows us to do?

Reply:
The intent was to use this to selectively skip scenarios that apply only to perf and only to stress. We would no longer need it if the two are separate suites.